src/Symfony/Component/Serializer/Tests/Normalizer/Features/CallbacksTestTrait.php

```diff
@@ -28,6 +33,85 @@ public function testCallbacks($callbacks, $valueBar, $result)
         );
     }
 
+    /**
+     * @dataProvider provideNormalizeCallbacks
+     */
+    public function testNormalizeCallbacksWithTypedProperty($callbacks, $valueBar, $result)
+    {
+        $normalizer = $this->getNormalizerForCallbacksWithPropertyTypeExtractor();
+
+        $obj = new CallbacksObject();
+        $obj->bar = $valueBar;
+
+        $this->assertEquals(
```

Comment:
on one line
if we can use asserSame, let's use it instead (maybe we can't, I didn't check)